components/has/production/kustomization.yaml

```diff
@@ -3,8 +3,14 @@ kind: Kustomization
 resources:
   - ../base
   - ../base/external-secrets
+
+configMapGenerator:
+- literals:
+  - GITHUB_ORG="redhat-appstudio-appdata"
```

Comment:
Just confirming the name of this env var.  Is it `GITHUB_ORG` or `GITOPS_ORG` which was in the deleted gitops-org-patch.yaml

Reply:
this is either a bug or a bug fix - good catch @kim-tsao